package.json

```diff
@@ -11,7 +11,7 @@
     "frontend": "react-scripts start",
     "build": "react-scripts build",
```

Comment:
can you reset this file to be what's on dev? seems to have some unintended changes

Reply:
the 2 lines from home.js were removed